utilities/tidal_data_to_ob.m

```diff
@@ -92,9 +92,10 @@
     end
     k = find(startsWith(string(const_t'),lower(const{j})));
     if isempty(k)
-       disp(['No tidal data in file for constituent ' const{j}])
-       keep(j) = false;  
-       continue
+        disp(['No tidal data in file for constituent ' const{j}])
+        obj.f15.opealpha(j).name = const{j};
```

Comment:
What’s use case for this?

Reply:
The use case I was doing is the following:
const = ["Z0" "major8"];

The Z0 is the zero frequency constituent that is actually valid in the constituent list but TPXO8 provides no data. So before I would just have this deleted from the boundary condition list. However, I was using Z0 to impose a sea level rise offset. 

So I was using Make_f15 to populate Z0 with "zeros" as a dummy into the boundary constituent list then overwrite it later with the desired amplitude. An additional thing I was thinking to add was in writefort15.m that it would only write it out if it wasn't all zeros to avoid cluttering of fort.15 but I think this could be considered a fairly minor concern unless you have many constituents.

Reply:
so in the case you're applying a SLR offset using this tidal constituent, you also add a geoid offset to the NA file?

Reply:
no I did not but I suppose you could (should?) do that as initial condition yes

Reply:
I was using small domain and just spun it up slowly to that state.

Reply:
yea I think you should specify an initial condition that matches the elevation of the boundary condition otherwise you'll create a large gradient near the boundary in free surface elevation.

but this for some more advanced users anyway.

Reply:
nah but since you spin it up from zero it works. And also even if elevation is below the new sea level with SLR it may not get flooded due to connectivity so that's why I just did the boundary condition